### ISSUE TITLE
References to old GH name

### ISSUE THREAD

**Issue by jakirkham** [gdrive-org/gdrive#461]
Am seeing a lot of references to the old GitHub name that this repo lived under. Here's [an example]( https://github.com/gdrive-org/gdrive/blob/c3cbcceedd6beb1fcff30f06ea7be7c29558d181/compare.go#L5 ). Seems to cause some issues when building from source. Would be nice to have them updated.

**Comment by mbenlioglu** [comment]
Yes, I'm aware of them, currently I'm refactoring the code locally, soon I'll update them on master branch as well
